modules/updating-eus-to-eus-upgrade.adoc

```diff
@@ -0,0 +1,144 @@
+Following this procedure reduces the total upgrade duration and the number of times workloads are restarted.
+
+.Prerequisites
+
+* Review the release notes for {product-title} 4.9 and 4.10
+* Ensure that you are familiar with version-specific prerequisites, such as Administrator Acknowledgement that is required prior to upgrading from {product-title} 4.8 to 4.9.
+* Verify that your cluster is not running {product-title} 4.8.14 version or earlier.
+If your cluster is running {product-title} 4.8.14 or earlier, you must upgrade to a later 4.8.z version prior to upgrading to 4.9.
+The upgrade to 4.8.14 or later fulfills the minimum version requirements that must be performed without pausing MachineConfigPools.
+* Verify that MachineConfigPools is unpaused.
+
+.Procedure
+
+. Verify that all MachineConfigPools display a status of `UPDATED` and no MachineConfigPools display a status of `UPDATING`.
+To view the status of all MachineConfigPools, run the following command:
++
+[source,terminal]
+----
+$ oc get mcp
+----
++
+.Example output
++
+Output is trimmed for clarity:
++
+[source,terminal]
+----
+NAME 	   CONFIG                                         	   UPDATED UPDATING
+master   rendered-master-ecbb9582781c1091e1c9f19d50cf836c    True  	 False
+worker   rendered-worker-00a3f0c68ae94e747193156b491553d5    True  	 False
+----
+
+. To pause the MachineConfigPools you wish to skip reboots on, run the following commands:
++
+[NOTE]
+====
+You cannot pause the master pool.
+====
++
+[source,terminal]
+----
+$ oc patch mcp/worker --type merge --patch '{"spec":{"paused":true}}'
+$ oc get mcp -o
+----
++
+.Example output
++
+[source,terminal]
+----
+custom-columns=NAME:.metadata.name,PAUSED:.spec.paused
+NAME 	   PAUSED
+master   false
+worker   true
+----
+
+. To change to the `eus-4.10` channel and upgrade to 4.9, run the following commands:
++
+[source,terminal]
+----
+$ oc adm upgrade channel eus-4.10
+$ oc adm upgrade --to-latest
+----
++
+.Example output
++
+[source,terminal]
+----
+Updating to latest version 4.9.11
+----
+
+. To ensure the 4.9 upgrades is completed successfully retrieve the cluster version, run the following command:
++
+[source,terminal]
+----
+$ oc get clusterversion
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME  	  VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
+version   4.9.11	  True    	  False     	  6m29s   Cluster version is 4.9.11
+----
+
+. To upgrade to 4.10, run the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade --to-latest
+----
+
+. To ensure the 4.10 upgrade is completed successfully retrieve the cluster version, run the following command:
++
+[source,terminal]
+----
+$ oc get clusterversion
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME  	  VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
+version   4.10.1	  True    	  False     	  6m29s   Cluster version is 4.10.1
```

Comment:
@sdodson - Could you verify this example output? There wasn't any example output in the Google Doc, but I based it off the 4.9.11 above.

Reply:
That's fine.